### PR TITLE
move load_file and example_dag tests into their own section

### PR DESCRIPTION
**Pull request by dimberman**
# Description

In trying to optimize the astro SDK CI system, I find that the load_file and example_dag sections are two sections that just can NOT be optimized using the mocking library. The load_file library can not be optimized because we are literally testing whether the files are being properly loaded into the external databases, and the example DAGs need to run exactly to spec so we are sure users can run them as-is.

To handle this limitation while still bringing down the total CI time, we separate the load_file tests and example_dags tests into their own section where we can split them using pytest-split and run them in parallel (where before they would run sequentially on a single worker).

We can now run all load_file tests within 10 minutes across three workers. and example_dag tests in 12 minutes.

### Checklist
- [ ] Created tests which fail without the change (if possible)
- [ ] Extended the README / documentation, if necessary

### CODE REVIEW

File: python-sdk/.test_durations [hunk 1]
```diff
@@ -0,0 +1,70 @@
+{
```
Comment (tatiana) [inline python-sdk/.test_durations:1]: @dimberman are we committing this file intentionally?

**Review by tatiana — COMMENTED** [review]
LGTM, @dimberman , one minor comment

File: python-sdk/.test_durations (hunk 1 above)
Comment (dimberman) [reply python-sdk/.test_durations:1]: @tatiana yes that file is used by pytest-split to properly balance the tests

File: python-sdk/tests_integration/sql/operators/test_load_file.py [hunk 2]
```diff
@@ -515,7 +505,7 @@ def test_load_file_chunks(sample_dag, database_table_fixture):
         ),
     ],
     indirect=["database_table_fixture"],
-    ids=["Bigquery", "Redshift"],
+    ids=["Redshift"],
```
Comment (tatiana) [inline python-sdk/tests_integration/sql/operators/test_load_file.py:508]: Why did we remove BQ?

Comment (sunank200) [reply python-sdk/tests_integration/sql/operators/test_load_file.py:508]: +1

**Review by sunank200 — APPROVED** [review]

File: python-sdk/tests_integration/sql/operators/test_load_file.py (hunk 2 above)
Comment (dimberman) [reply python-sdk/tests_integration/sql/operators/test_load_file.py:508]: @tatiana this one test took 5 minutes to complete and as far as I can tell it's not really testing anything that's not already tested (note that code coverage has not changes at all)

File: python-sdk/.test_durations (hunk 1 above)
Comment (feluelle) [reply python-sdk/.test_durations:1]: Nice! ❤️

**Pull request merged**